packages/astro/src/@types/astro.ts

```diff
@@ -462,6 +462,24 @@ export interface AstroUserConfig {
 	 */
 	site?: string;
 
+
+	/**
+	 * @docs
+	 * @name compressHTML
+	 * @type {boolean}
+	 * @default `true`
```

Comment:
```suggestion
	 * @default `false`
```

The RFC says that by default the option should be `false`